.travis.yml

```diff
@@ -25,7 +25,10 @@ script:
   - docker-compose run --rm webtest "rspec"
   - docker-compose run --rm webtest "npm test"
   - docker-compose run --rm webtest "pip install mock; python -m unittest discover -v -s /app/test/ -p 'test_*.py'"
+  # Run brakeman once to get a full report.
```

Comment:
Do we still need this?

Reply:
I was thinking it'd be easier for people to notice their Low/Medium warnings. but I also doubt people check it so.. 👍/👎?

Reply:
ok I just removed the full run. Non-actioning reports don't really belong in CI